binaries/data/mods/public/gui/prelobby/common/terms/Privacy_Policy.txt

```diff
@@ -41,13 +42,37 @@ The "service" is the 0 A.D. Empires Ascendant Multiplayer Lobby provided by Wild
-Usernames are chosen freely on this service, can differ from or collide with names of other users on other services by Wildfire Games such as the Forums.
-Wildfire Games ask you to not publish personal data on this service, especially if you are below 16 years of age.
-User submitted content on the service is public, not verifiable to us and does not enable us to identify your natural person.
-Unless your data is stored for a legal obligation such as Abuse, your userdata will become anonymous if you stop using this service.
-
-[font="sans-bold-14"]If you wish to file personal data request in accordance with the General Data Protection Regulation:[/font]
-Please write an email to user-data@wildfiregames.com.
-You will have to authenticate yourself with the service to prevent abuse of personal data requests.
+The IP address is the only information processed that could reliably identify your natural person. (GDPR 4.5)
+Usernames are chosen freely on this service, can differ from or collide with names of same or other users on other services run by Wildfire Games such as the Forums.
+User submitted content on the service is public, not verifiable to us and, according to the Terms of Use, must not enable us to identify your natural person.
+You will have to authenticate yourself with the service to enforce your rights (GDPR 12.6).
+
+[font="sans-bold-14"]Personal information disclosure:[/font]
+As a rule of thumb, Wildfire Games asks you to not publish personal data (real names, identification number, location data, discriminating factors, etc.) other than usernames on this service, especially if you are below 16 years of age.
+In case personal information about your own natural person was disclosed in the Lobby Chat, Wildfire Games will comply with your right to erasure and will edit out the data from the logs upon request. However, all users who were logged in the Chat at the time of disclosure will receive the information and you will have to enforce your rights to them if they wish to process it. Additionally:
+ 1. If another user disclosed the information, they will be banned for the service for violating the Terms of Use.
+ 2. If you disclosed the information yourself, Wildfire Games will comply with your request, but you will face a ban if you abusingly and repetitively disclose information that you wanted erased.
+
+[font="sans-bold-14"]Legal grounds of processing (GDPR 13.1.c):[/font]
+1. The processing is necessary for the performance of the service defined in this document (GDPR 6.1.b).
+2. Wildfire Games has legitimate interests in providing the service, including Playerlist, Lobby Chat, Gamelist, Statistics, Rating and Ranking, as part as its entertainment reason for being. Wildfire Games has legitimate interests in processing Feedback messages in order to further the development of 0 A.D. Wildfire Games has legitimate interests in providing the Moderation service to defend itself and its users against cyberattacks (GDPR 6.1.f).
```

Comment:
Providing the lobby is equally as legitimate as processing all the feedback we inject into creating the game. We provide 0 A.D. and its lobby for entertainment.